src/tools/ci/docker/linux_fedora_rawhide

```diff
@@ -0,0 +1,49 @@
+FROM fedora:rawhide
+RUN \
+  yum -y update && \
+  yum -y install \
+  git \
+  meson \
+  perl \
+  perl-IPC-Run \
+  \
+  bison \
+  ccache \
+  clang \
+  flex \
+  gcc \
+  gettext \
+  lz4 \
+  ninja-build \
+  \
+  libicu-devel \
+  libuuid-devel \
+  libxml2-devel \
+  libxslt-devel \
+  llvm-devel \
+  lz4-devel \
+  pam-devel \
+  python3-devel \
+  readline-devel \
+  systemd-devel \
+  tcl-devel \
+  \
+  openldap-devel \
+  openldap-servers \
+  openldap-clients \
+  \
+  openssl \
+  openssl-devel \
+  \
+  krb5-devel \
+  krb5-server \
+  krb5-server-ldap \
+  krb5-workstation \
+  \
+  langpacks-en \
+  glibc-langpack-en
+RUN \
+  # Meson tries to run typemap from /usr/share/perl5/vendor_perl/ExtUtils/typemap but typemap is
+  # at /usr/share/perl5/ExtUtils/typemap path
+  find /usr/share/perl5/ExtUtils -name typemap -exec ln -sf '{}' /usr/share/perl5/vendor_perl/ExtUtils/typemap ';'
```

Comment:
Hm, I think this is a bug in the meson build definition. Need to fix that, instead of adding workarounds - the point of testing on more platforms is precisely to find those :)

I assume the configure based build doesn't need it? What do we end with as perl_privlibexp in src/Makefile.global?

Reply:
It is `perl_privlibexp		= /usr/share/perl5`

Reply:
It should be fixed now. 
Error was: `subppdir` and `privlibexpdir` paths were different. I added a path for `privlibexpdir` and set it to used by typemap.